ClientSecurity/Utilizing_SLACK_for_IPS_Event_Notifications.md

```diff
@@ -0,0 +1,57 @@
+# Utilizing SLACK for IPS Event Notifications 
+
+### Overview
+
+Slack is a team collaboration tool that offers persistent chat rooms organized by topic and also has private groups and direct messaging capabilities. Slack is available as long as you want and can be utilized with an unlimited number of people.
+
+Our Intrusion Prevention System (IPS) provides real-time event notifications that can be integrated with Slack messaging. Using the Slack integration for “Incoming WebHooks”, our IPS can be set up to send event details on the rule triggered, source and destination information to a Slack channel so it can be quickly reviewed by your team or archived for review at a later time.
+
+### Signing Up for Slack
+Utilize the below link to sign up and get started with Slack!
+[https://slack.zendesk.com/hc/en-us/articles/206480347-Getting-started-with-Slack](https://slack.zendesk.com/hc/en-us/articles/206480347-Getting-started-with-Slack)
+
+### Creating a Slack Channel
+Utilize the below link to set up a specific Slack channel to receive IPS Event Notifications.
+[https://slack.zendesk.com/hc/en-us/articles/201402297-Creating-a-channel](https://slack.zendesk.com/hc/en-us/articles/201402297-Creating-a-channel)
+
+### Creating Webhook URL
+Setting up a WebHook will allow the IPS Event Notifications to be posted directly into your chosen Slack channel.
+
+1.)	Once your IPS Event Notifications Slack channel has been designated, you will need to click on the channel name drop-down and select “**Add a service Integration…**”
+![Webhook URL.png](https://ucarecdn.com/1d3c452e-18a7-4bf2-a667-408cf75e4546/)
+
+2.)	Scroll to the bottom of the Integrations page and select "**Incoming WebHooks**”.
+![DIY Integrations and Customizations.png](https://ucarecdn.com/96a2b388-7064-4817-a31c-7b44f7317d0c/)
```

Comment:
probably should not be linking to external image services for our own KB. push images into our repository